docs/.vuepress/theme/layouts/Layout.vue

```diff
@@ -4,7 +4,8 @@
       <template #page-top>
         <div v-show="show" class="page-top">
           <div class="version-alert">
-            <p>This page covers a non-latest version of Handsontable.</p>
+            <p>There is a newer version available.
+              <a href="/docs/latest/">Switch to the latest version</a>.</p>
```

Comment:
```suggestion
            <p>There's a newer version of Handsontable's documentation.
              <a href="/docs/latest/">Go to the latest version</a>.</p>
```

A newer version of what – this page, this site, this documentation, this feature, Handsontable?

Reply:
It's about Handsontable. We don't make versions of the documentation, at least not officially.

Reply:
I don't want that custom box to stand out. It might be annoying for someone who consciously previews the legacy version of Handsontable.

<img width="1426" alt="Screenshot 2022-10-14 at 09 08 53" src="https://user-images.githubusercontent.com/8048526/195784580-8614af20-4d5d-4393-8b5c-64cc0f4e8302.png">

<img width="1420" alt="Screenshot 2022-10-14 at 09 08 48" src="https://user-images.githubusercontent.com/8048526/195784588-e4b16c15-4969-472c-9827-5105cf8f0814.png">